### PR TITLE
Updating README.rst: Logging configuration

### PR DESCRIPTION
**Pull request by bquenin**
Explaining how to setup SLF4j as default logging framework with crate client.

### CODE REVIEW

**Comment by bquenin** [comment]
Documentation for #2591.

File: client/README.rst [hunk 1]
```diff
@@ -90,3 +90,14 @@ these, one must defined it at the `SQLRequest` object::
...
 
+Logging
+=======
+
+Crate client relies on Elastic Search ESLogger facility which itself defaults to `log4j` framework.
+
+Using Slf4j
+-----------
+
+In order to configure the create client to use Slf4j, you need to configure the ESLoggerFactory to use Slf4j by using the following command (typically in your entry point)::
```
Comment (celaus) [inline client/README.rst]: Did you mean 'Crate' client? :)

Comment (bquenin) [inline client/README.rst]: Maybe :)

**Comment by celaus** [comment]
Hi, thanks for your PR! Before we can merge, please squash and rebase your commits (https://gist.github.com/chaudum/d7bdafb0c478a6639f1a) into a single commit and sign our contributor license agreement: https://crate.io/community/contribute/agreements/ .

Thanks! 

Cheers, Claus

**Comment by bquenin** [comment]
I think it's good

**Comment by celaus** [comment]
Great, thanks. We will merge in a bit.

**Pull request closed without merging**